src/types/calls.ts

```diff
@@ -34,4 +34,8 @@ export type CallsLinkClickMessage = {
     link: string | URL;
 }
 
+export type CallsJoinRequestMessage = {
+    targetID: string;
```

Comment:
What's this target exactly? Is it the call?

Reply:
Yep. For now it's the channelID, but could change. Same as we've used targetID elsewhere.

Reply:
Interesting, I don't recall using `targetID` ever in the context of calls. I was expecting `callID` I guess. Is there a future this won't be accurate?

Reply:
Yeah, I think it should be `callID` on both sides ideally. The existing one on the calls side is only for internal use I suppose but here we are making a contract so I'd like to give it more thought and it seems like what we really mean is the id of the call (which can coincide with the channel ID but that's another matter) so we should name it accordingly.

Reply:
Should be good to go now, and the calls PR is updated.